src/auxinfo/info.rs

```diff
@@ -21,9 +21,12 @@ use tracing::{error, instrument};
 
 /// The private key corresponding to a given Participant's [`AuxInfoPublic`].
 ///
-/// TODO #169: Let's be more careful about allowing `Clone`, `Serialize`, etc.
+/// # ⚠️ Storage requirements
+/// This type must be stored securely by the calling application.
+///
+/// TODO #169: Let's be more careful about allowing `Clone`, etc.
```

Comment:
FYI this issue no longer exists, and it appears we don't need `Serialize` and `Deserialize` anyways. Should I remove this note?

Reply:
Yes please. I'm not even sure how this related to the former #169.